src/libstd/process.rs

```diff
@@ -40,11 +40,15 @@ use thread;
 /// ```should_panic
 /// use std::process::Command;
 ///
-/// let output = Command::new("/bin/cat").arg("file.txt").output().unwrap_or_else(|e| {
```

Comment:
The example is on http://doc.rust-lang.org/nightly/std/process/struct.Child.html but never used the Child type, so I changed to spawn which does.